src/views/PasswordResetView.vue

```diff
@@ -0,0 +1,226 @@
+        <span class="p-float-label with-error">
+          <InputText v-bind="password2" id="password2" size="large" name="password" type="password" :class="{'p-invalid': passwordForm.errors.value.password2}"/>
+          <label :class="{'contains-text': password2.modelValue }" for="password2">{{ $t('login.Confirm password') }}</label>
+        </span>
+        <small v-if="passwordForm.errors.value.password2" class="p-error"><i class="pi pi-exclamation-circle"/>{{ passwordForm.errors.value.password2 }}</small>
+        <Button type="submit" id="reset-button" severity="danger">{{ $t('login.Reset') }}</Button>
+        <div class="backtologin" @click="backToLogin">{{ $t('login.Back to login') }}</div>
+      </Form>
+    </main>
+    <CopyrightBanner/>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Form } from 'vee-validate';
+import CopyrightBanner from "@/components/CopyrightBanner.vue";
+import apiService from "@/services/ApiService";
+import router from "@/router";
+import { useForm } from "vee-validate";
+import InputText from "primevue/inputtext";
+import { onBeforeMount, ref } from "vue";
+import { useRoute } from "vue-router";
+import * as yup from "yup";
+import { toTypedSchema } from "@vee-validate/yup";
+
+const emailSchema = toTypedSchema(
+    yup.object({
+      email: yup.string().email().required(),
+    })
+);
+
+const passwordSchema = toTypedSchema(
+    yup.object({
+      password: yup.string().required(),
+      password2: yup.string().required(),
+    })
+);
+
+const emailForm = useForm({
+  validationSchema: emailSchema,
+});
+
+const passwordForm = useForm({
+  validationSchema: passwordSchema,
+});
+
+const passwordResetMode = ref(0);
+const email = emailForm.defineComponentBinds('email');
+const password = passwordForm.defineComponentBinds('password');
+const password2 = passwordForm.defineComponentBinds('password2');
+
+const route = useRoute();
+
+onBeforeMount(async () => {
+  if (route.query.token !== undefined && route.query.email !== undefined) {
+    const token = route.query.token as string;
+    const email = route.query.email as string;
+    passwordResetMode.value = 2;
+    console.log(token);
+    console.log(email);
+  }
+});
+
+const resetPasswordRequest = emailForm.handleSubmit(async () => {
+  await apiService.authenticate.resetLocal({ accountMail: email.value.modelValue })
+      .then(() => {
+        passwordResetMode.value = 1;
+      });
+});
+
+const setNewPassword = passwordForm.handleSubmit(async () => {
+  console.log(route.query.email);
+  console.log(route.query.token);
+  console.log(password.value.modelValue);
+
+  await apiService.authenticate.resetLocalWithToken({
+    accountMail: route.query.email as string,
+    token: route.query.token as string,
+    password: password.value.modelValue as string,
+  }).then();
+});
+
+
+
+
+const backToLogin = () => {
+  router.push({ name: 'login' });
+};
+
+</script>
+
+<style scoped lang="scss">
+//TODO Cleanup and fix

```

Comment:
Please add a link to #14.

Reply:
Added TODO's for #14 and #25 in PasswordResetView, also updated the description to include #24